src/pages/images-constructor/tests/images-constructor.spec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { ThemeProvider } from '@material-ui/styles';
+import { theme } from '../../../components/app/app-theme/app.theme';
+import ImagesConstructor from '../images-constructor';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const themeValue = theme('light');
+
+let wrapper;
+
+jest.mock('horondi_merge_images', () => ({ DEFAULT_PRICE_VALUE: '1400' }));
+
+describe('ImagesConstructor component tests', () => {
+  wrapper = shallow(
+    <ThemeProvider theme={themeValue}>
```

Comment:
Mock useStyles and then you can remove ThemeProvider. Thanks.

Reply:
applied for constructor-submit.spec.js , in current file this causes issues with wrapping with <Provider>, can't find solution